packages/zoe/test/unitTests/contracts/test-sellTickets.js

```diff
@@ -507,8 +507,6 @@ test(`mint and sell opera tickets`, async t => {
 
     const operaPurse = moolaIssuer.makeEmptyPurse();
 
-    await E(sellItemsCreatorSeat).tryExit();
```

Comment:
huh, is this removed to test the functionality? can we add a comment as such?

Reply:
I removed it because the seat has been exited by this time, so it's not needed, and causes an extraneous error.

Reply:
Ok sounds good